.circleci/config.yml

```diff
@@ -454,7 +454,44 @@ jobs:
             echo "export DOCKER_BUILDKIT=" > $BASH_ENV
             force_deploy_build aztec-sandbox false arm64
 
-  aztec-sandbox-ecr-manifest:
+  pxe-base:
+    machine:
+      image: ubuntu-2204:2023.07.2
+    resource_class: large
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Build and test"
+          command: force_deploy_build pxe-base false
+
+  pxe-x86_64:
+    machine:
+      image: ubuntu-2204:2023.07.2
+    resource_class: large
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Build and test"
+          command: force_deploy_build pxe false x86_64
```

Comment:
This script is abot to be removed. You will need to check the above PR to see the new pattern of arch specific builds